src/runner.ts

```diff
@@ -84,4 +84,7 @@ const runWithStartCommand = async (serveCmd: string, serveHost: string, runCypre
 	}
 };
 
-export const runHandler = dockerize(runCommand);
+export const runHandler = (argv: Arguments<ArgTypes>) => {
+	loadConfig(argv);
+	dockerize(runCommand)();
```

Comment:
Now we kind of break the original functional style which used function composition, and that's a bit counter-intuitive now...

Reply:
See my new commit, which addresses the problem. I've removed the use of the function composition technique, as I didn't find it beneficial for this case. The `runCommand` is simply not called (in the same script invocation) if we use Docker.